hkm/views/views.py

```diff
@@ -643,7 +690,7 @@ def handle_add_to_collection(self, request, *args, **kwargs):
         record.save()
 
         url = reverse('hkm_search_record')
-        url += '?search=%s&page=%d' % (self.search_term, self.page)
+        url += '?search=%s&page=%d' % (self.url_params['serach'], self.url_params['page'])
```

Comment:
"serach" is a typo in our end I guess?

Reply:
Thats a typo on my part, nicely spotted 👍